storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -926,10 +926,10 @@
     public static final String UI_HTTPS_NEED_CLIENT_AUTH = "ui.https.need.client.auth";
 
     /**
-     * The host that Pacemaker is running on.
+     * The list of servers that Pacemaker is running on.
      */
-    @isString
-    public static final String PACEMAKER_HOST = "pacemaker.host";
+    @isStringList
+    public static final String PACEMAKER_SERVERS = "pacemaker.servers";
```

Comment:
This is a breaking change that would be good to go back to 1.x as well.  I am OK with this, because I don't think anyone is really using pacemaker, but I would like to hear others opinions on it.